assets/app/components/site/editor/editorContainer.js

```diff
@@ -22,7 +22,7 @@ const propTypes = {
 };
 
 const redirectToFileOnBranch = (siteId, branch, filePath) => {
-  routeActions.redirect(`/sites/${siteId}/edit/${branch}/${filePath}`);
+  routeActions.redirect(`/sites/${siteId}/edit/${branch}/${filePath}`, 'replace');
```

Comment:
Whatever you think the proper names to communicate those concepts to callers to the methods works for me!